src/utils/http/errors/NotFoundApiError.js

```diff
@@ -0,0 +1,15 @@
+const httpStatusCodes = require('../httpStatusCodes');
+const BaseError = require('./ApiError');
+
+class NotFoundApiError extends BaseError {
```

Comment:
Why does this extend BaseError not just Error?

Reply:
Why renaming `ApiError` to `BaseError`?

Reply:
not intentional, fixed